suite-common/test-utils/src/setupJest.ts

```diff
@@ -3,12 +3,18 @@
 /* eslint-disable require-await */
```

Comment:
I think a lot about how solve this correctly and I don't like with using some global Jest mock object. Instead of this strange stuff why not simply imports mock like all other stuff?

Please move all mocks to file called `mocks` and normally export it. Then move rest of the file back to `packages/suite` (it will be just setup of that global mocks)  and import mock there. We will use this global jest stuff only here for because we don't want to migrate everything in suite.

Everywhere else in new packages, just import mocks using standart import. `import {mocks} from '@suite-common/test-utils'`

And plese update this package README.md according to it.

Sorry for this change, but this is much better approach that relying on some jest global stuff that is breaking TS and also possibly hardening migration to some other solution like Vitest which we may want to use in future because it's much faster.